alvr/server_openvr/cpp/platform/win32/VideoEncoderAMF.cpp

```diff
@@ -709,7 +709,8 @@ void VideoEncoderAMF::Transmit(
 
     auto params = GetDynamicEncoderParams();
     if (params.updated) {
-        amf_int64 bitRateIn = params.bitrate_bps / params.framerate * m_refreshRate; // in bits
```

Comment:
This should probably be handled via clamping (with a warn/info message) instead, especially because I think it'd be useful to be able to turn up the bitrate even higher in the dashboard for encoders that support it.